tensorboard/webapp/widgets/line_chart_v2/sub_view/line_chart_axis_view.scss

```diff
@@ -92,3 +104,53 @@ text {
     );
   }
 }
+
+.extent-edit-button {
+  background-color: mat-color($mat-gray, 200);
+  display: none;
+  font-size: 0;
+  height: 24px;
+  line-height: 24px;
+  margin: 5px;
```

Comment:
nit: setting margin together with pos:abs seems a bit redundant, maybe just specify the right and drop margin?  e.g. `right: 5px`